components/Footer.jsx

```diff
@@ -9,45 +8,22 @@ const getCurrentYear = () => {
 
 const Footer = () => {
   return (
-    <Grid
-      container
-      direction="row"
-      justifyContent="space-between"
-      alignItems="center"
-      sx={{
-        width: "100%",
-        height: "auto",
-        padding: "1rem",
-        bgcolor: "primary.main"
-      }}>
-      <Grid item xs={12} sm={6} container justifyContent="flex-start" alignItems="center">
-        {/* Copyright text */}
-        <Typography
-          variant="body2"
-          color="background.paper"
-          marginLeft="1rem">{` © ${getCurrentYear()}  PLAYTRADE`}</Typography>
-      </Grid>
-      <Grid item xs={12} sm={6} container justifyContent="flex-end" alignItems="center">
-        {/* List of links */}
-        <ul style={{ listStyle: "none", margin: 0, display: "flex" }}>
-          <li style={{ marginRight: "1rem" }}>
-            <Link href="/team" style={{ textDecoration: "none", color: "#FFFFFF" }}>
-              Our Team
-            </Link>
-          </li>
-          <li style={{ marginRight: "1rem" }}>
-            <Link href="/policies" style={{ textDecoration: "none", color: "#FFFFFF" }}>
-              Policies
-            </Link>
-          </li>
-          <li style={{ marginRight: "1rem" }}>
-            <Link href="/messages" style={{ textDecoration: "none", color: "#FFFFFF" }}>
-              Contact Us
-            </Link>
-          </li>
-        </ul>
-      </Grid>
-    </Grid>
+    <>
+      <Box sx={{ display: "flex", alignItems: "center", p: 2.5, gap: 2, bgcolor: "primary.main" }}>
+        <Typography variant="body2" color="background.paper">{` © ${getCurrentYear()}  PLAYTRADE`}</Typography>
+        {/* Spacer */}
+        <Box sx={{ flexGrow: 1 }} />
+        <Link href="/team" style={{ textDecoration: "none", color: "background.paper" }}>
```

Comment:
Hi Luba! Thank you for MUI in our component.

Link here is not MUI component, so it doesn't see our theme such easy in styles. I mention that this code you are added before, disappeared from out global.css:
`a,
a:link,
a:visited,
a:focus,
a:hover,
a:active {
  color: #FFFFFF;
  text-decoration: none;
  cursor: crosshair;
}
`
we need to fix this part: 
<img width="342" alt="image" src="https://github.com/Code-the-Dream-School/ffprac-team6/assets/106414157/c2a9cc38-242a-42ed-89fb-e66741329ff6">